engine/gameobject/src/gameobject/gameobject.cpp

```diff
@@ -259,7 +259,24 @@ namespace dmGameObject
         delete regist;
     }
 
-    Collection* AllocCollection(const char* name, HRegister regist, uint32_t max_instances)
+    uint32_t GetMaxComponentIntances(uint64_t name_hash, dmGameObjectDDF::CollectionDesc* collection_desc)
+    {
+        if (!collection_desc || collection_desc->m_ComponentTypes.m_Count == 0)
```

Comment:
We need that for back compatibility with old formats or manual creation of collection we use in our tests

Reply:
"GetMaxComponentIn***s***tances" 🙈

Reply:
  🙉
  thx!)